app/src/main/cpp/moz_external_vr.h

```diff
@@ -133,11 +133,15 @@ enum class ControllerCapabilityFlags : uint16_t {
 
 enum class VRControllerType : uint8_t {
   _empty,
-  Vive,
-  WMR,
-  Knuckles,
-  Cosmos,
+  HTCVive,
+  HTCViveCosmos,
+  HTCViveFocus,
+  HTCViveFocusPlus,
+  MSMR,
+  ValveIndex,
+  OculusGo,
   OculusTouch,
+  OculusTouch2,
```

Comment:
Are we treating the Pico controllers like Quest and Go controllers?

Reply:
Sure, I will take a look at PicoDelegate to understand how diverse controller we have.